### PR TITLE
Send email in same flow as text message when meeting starts

### PR DESCRIPTION
**Pull request by sent-hil**
# TODOS:
- [x] Update Readme to include how to setup AWS SES.
- [ ] Update email body with starter template.
- [x] Merge https://github.com/codeforsanjose/gov-agenda-notifier/pull/100 first

### :scroll: Description

- Describe the general shape of this PR (new feature? refactor? bug fix? one-line change?)
Send email in the same flow as SMS when the meeting starts. Closes https://github.com/codeforsanjose/gov-agenda-notifier/issues/97

- Describe what changes are being made
Creates sesClient which sends email through AWS SES.

- Describe how errors will be handled. How will we know if this code breaks in production
Errors due to AWS SES API are logged using `logger`.

- Describe any external libraries/dependencies added or removed in this PR
aws-sdk added.

- Describe any security risks are there regarding this change
None.

- Describe how you tested these changes
Made this change on graphql API frontend and saw email was sent.
```
mutation {
  updateMeeting(
    id: 1,
    status: "IN PROGRESS",
    meeting_start_timestamp: "10",
    meeting_end_timestamp: "10",
    virtual_meeting_url:"https://google.com",
    meeting_type:"test"
  ) {
  	status
  }
}
```

- Link to relevant external documentation

--------------------------
### :clipboard: Mandatory Checklist
- [ ] Linked to the Github Issues being addressed using the right sidebar :arrow_right:
- [x] Have you discussed these changes with the project leader(s)?
- [x] Do all variable and function names communicate what they do?
- [x] Were all the changes commented and / or documented?
- [x] Is the PR the right size? (If the PR is too large to review, it might be good to break it up into multiple PRs.)
- [x] Does all work in progress, temporary, or debugger code have a TODO comment with links to Github issues?
- [x] *If you changed the user interface, did you add before and after screenshots to below?*

--------------------------
#### After

image

--------------------------
### :blue_book: Glossary
- PR = Pull Request

### CODE REVIEW

File: backend/graphql_api/lambda/controllers/subscriptionsController.js [hunk 1]
```diff
@@ -1,22 +1,25 @@
 const getTwilioClient = require('../twilio/twilioClient');
+const getSesClient = require('../ses/sesClient');
 
+// TODOS: who calls this?
```
Comment (teohrt) [inline backend/graphql_api/lambda/controllers/subscriptionsController.js:4]: My apologies for that export pattern. I wasn't yet familiar with any javascript dependency injection frameworks so this was my manual solution. 

Searching the codebase for the filename is a decent strategy in this situation though. The `requires` will pop up.

It's used in the mutation resolvers here: 
https://github.com/codeforsanjose/gov-agenda-notifier/blob/6c7baa866506089ce58d078b34023771da5f3b4d/backend/graphql_api/lambda/graphql/resolvers/mutation.js#L4

Don't forget to remove this question if you're happy with the answer

**Review by teohrt — CHANGES_REQUESTED** [review]
Awesome! Just a few things to shore up but this is great.

In addition to my other comments, I love that docker-compose flow. Can you update the README with some documentation on it before it hits master?

**Comment by sent-hil** [comment]
@teohrt my bad for leaving those random todos, i was just trying to understand the codebase and accidentally pushed them 😬

File: backend/graphql_api/lambda/controllers/subscriptionsController.js [hunk 2]
```diff
@@ -106,6 +132,7 @@ module.exports = (logger) => {
     }
   };
 
+  // TODOS: this notifies users meeting itself is starting
```
[the remark this answers is not in the document]
Comment (sent-hil) [reply backend/graphql_api/lambda/controllers/subscriptionsController.js]: removed it

**Review by JMStudiosJoe — APPROVED** [review]
looks great

**Pull request merged**